src/freenet/support/Logger.java

```diff
@@ -1,206 +1,17 @@
 package freenet.support;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.lang.ref.WeakReference;
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
-import java.nio.charset.StandardCharsets;
-import java.util.regex.PatternSyntaxException;
 
 import freenet.support.FileLoggerHook.IntervalParseException;
 import freenet.support.LoggerHook.InvalidThresholdException;
-import freenet.support.io.Closer;
 
 /**
  * @author Iakin
 
 */
 public abstract class Logger {
-	public final static class OSThread {
```

Comment:
used in SCGIPublisher ⇒ can’t just remove.

Reply:
⇒ we need a shim (that doesn’t do anything).

Reply:
No sane plugin^W^W^W Restored the Logger$OSThread methods, but removed their implementation.
Thanks for catching!

Reply:
Thank you! SCGIPublisher is actually sane and useful (hosting hyphanet inproxies with strict allow-list), but it is also old and unmaintained.